core/components/organisms/textField/TextField.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import TextFieldWithTextarea, { TextFieldTextareaProps } from './TextFieldWithTextarea';
+import TextFieldWithInput, { TextFieldInputProps } from './TextFieldWithInput';
+
+export type TextFieldProps = TextFieldTextareaProps & TextFieldInputProps;
+
+export const TextField = (props: TextFieldProps) => {
+  return props.withTextarea ? <TextFieldWithTextarea {...props} /> : <TextFieldWithInput {...props} />;
+};
```

Comment:
Instead of using ternary operator please use the following: 

```
export const TextField = (props: TextFieldProps) => {
  const { withTextarea } = props;

  if (withTextarea) {
    return <TextFieldWithTextarea {...props} />;
  }

  return <TextFieldWithInput {...props} />;
};

```